lib/styles.less

```diff
@@ -4,6 +4,8 @@ body {
     padding: 0;
 
     background: @style_body_background;
+
+    overflow-y: hidden;
```

Comment:
was this intentional? we can't leave this in, as it would break scrolling

Reply:
Rolled back to preserve iframe scrolling